channels/chan_echolink.c

```diff
@@ -1820,6 +1825,23 @@ static void send_info(const void *nodep, const VISIT which, const int depth)
 			ast_log(LOG_WARNING, "Exceeded buffer size");
 			return;
 		}
+
+		if (instp->mymessage[0] != '\0') {
```

Comment:
Add a comment in English explaining what you are trying to do here (convert all the "\\n" to "\n")
Also, I'm curious why you choose to modify the original buffer here, it seems like the memmove's will therefore only happen the first time? In that case why not choose to do that when the buffer is initialized to begin with? That might be a little bit more logical. Alternately, if you wanted to leave the original buffer intact, you could move the snprintf calls into the loop.